FindmallocMC.cmake

```diff
@@ -97,44 +102,47 @@ if(mallocMC_ROOT_DIR)
     # mallocMC variables ########################################################
     #
     set(mallocMC_VERSION "${mallocMC_VERSION_MAJOR}.${mallocMC_VERSION_MINOR}.${mallocMC_VERSION_PATCH}")
-    set(mallocMC_INCLUDE_DIRS ${mallocMC_ROOT_DIR}/include)
+    set(mallocMC_INCLUDE_DIRS ${mallocMC_ROOT_DIR}/include CACHE PATH "mallocMC include pathes")
 
     # check additional components ###############################################
     #
     foreach(COMPONENT ${mallocMC_FIND_COMPONENTS})
-        set(mallocMC_${COMPONENT}_FOUND TRUE)
+        set(mallocMC_${COMPONENT}_FOUND TRUE CACHE BOOL
+            "The mallocMC component ${COMPONENT} exists in the found installation.")
 
         if(${COMPONENT} STREQUAL "halloc")
 
             # halloc linked library #################################################
             #
             list(APPEND mallocMC_REQUIRED_VARS_LIST mallocMC_LIBRARIES)
-            find_library(${COMPONENT}_LIBRARY
+            find_library(mallocMC_${COMPONENT}_LIBRARY
```

Comment:
Should there also be a preliminary call to `find_library` that uses `NO_DEFAULT_PATH`?
And the second call of course without the `PATHS` option.